assets/scss/_grpc.scss

```diff
@@ -44,6 +49,17 @@ sections site wide to the original site's background svg.
   display: none;
 }
 
+// Override Docsy theme padding values to reduce vertical space taken up by the hero section
```

Comment:
What is meant by `hero` section?
IIUC, the hero refers to the "gRPC" logo in the top left inside the navbar. The style below seems to push down the content of the `main` section so that they don't go below the `header` section (which now coantains the banner + navbar). If this is the case, we should call it the `header` section and not the `hero` section.

Reply:
Hero Section is an Introductory Section of Content that can be placed immediately before or after the Header. In our case 
![Screenshot 2024-07-22 12 26 36 PM](https://github.com/user-attachments/assets/656a03ba-a99c-4cbf-9ddc-ef151226912c)
This is the hero section.